<!-- wendao   -->
<launch>
	<node name="lane_detect_node" pkg="lane_detect" type="lane_detect.py" >
		<param name="is_debug" value="true" />
		<param name="is_reconfig" value="true" />
	</node>
	
	<node name="show_result" pkg="lane_detect" type="show_result.py" />
	
</launch>
